declarations.d.ts

```diff
@@ -228,3 +228,16 @@ interface ITypeScriptCompilationService {
 	initialize(typeScriptFiles: string[]): void;
 	compileAllFiles(): IFuture<void>;
 }
+
+interface IDynamicHelpService {
+	isProjectType(...args: string[]): IFuture<boolean>;
```

Comment:
As this method should be called from help.txt in the following way:
<% if (#{dynamicHelpService.isProjectType(cordova)}) { %> help for cordova <% } %>
or 
<% if (#{dynamicHelpService.isProjectType(cordova, nativescript)}) { %> help for cordova and nativescript <% } %>
the number of parameters is not defined. And in injector's dynamicCall we call
```
module[parsed[2]].apply(module, args);
```
where args are the passed arguments. In case I define isProjectType with (args:string[]) the first parameter(cordova in the example above) will be passed to args and rest of arguments will be omitted.